enterprise/cmd/frontend/internal/authz/resolvers/resolver.go

```diff
@@ -519,12 +522,17 @@ func getOrDefault[T any](ptr *T) T {
 }
 
 type permissionsInfoResolver struct {
+	db           edb.EnterpriseDB
+	ossDB        database.DB
```

Comment:
Consider renaming `db` to something better, since we apparently need both:
```suggestion
	eDB           edb.EnterpriseDB
	ossDB        database.DB
```